### PR TITLE
Use v3 company API

### PR DESCRIPTION
**Pull request by reupen**
Uses the back end's v3 company API instead of the old v1 company API.

This also fixes a bug editing companies where the trading address field would always be blank in the form.

Note: None of the pages that display a list of interactions have pagination – that needs addressing (separately).

### CODE REVIEW

File: src/apps/contacts/repos.js [hunk 1]
```diff
@@ -42,8 +42,14 @@ function unarchiveContact (token, contactId) {
 function getContactsForCompany (token, companyId) {
   return new Promise(async (resolve, reject) => {
     try {
-      const company = await authorisedRequest(token, `${config.apiRoot}/company/${companyId}/`)
-      resolve(company.contacts)
+      const response = await authorisedRequest(
```
Comment (teneightfive) [inline src/apps/contacts/repos.js]: Maybe this instead (keeps consistent with other uses):

```js
const response = await authorisedRequest(token, {
  url: `${config.apiRoot}/v3/contact`,
  qs: {
    company_id: companyId,
  },
})
```

**Comment by teneightfive** [comment]
👍

File: test/unit/apps/companies/controllers/ltd.test.js [hunk 2]
```diff
@@ -170,7 +170,7 @@ describe('Company controller, ltd', function () {
           expect(getDisplayCompanyStub).to.be.calledWith(company)
           expect(res.locals).to.have.property('companyDetails')
           expect(res.locals).to.have.property('companyDetailsLabels')
-          expect(res.locals.companyDetailsDisplayOrder).to.deep.equal(['alias', 'trading_address', 'uk_region', 'headquarter_type', 'sector', 'website', 'description', 'employee_range', 'turnover_range'])
+          expect(res.locals.companyDetailsDisplayOrder).to.deep.equal(['trading_name', 'trading_address', 'uk_region', 'headquarter_type', 'sector', 'website', 'description', 'employee_range', 'turnover_range'])
```
Comment (tyom) [inline test/unit/apps/companies/controllers/ltd.test.js]: Maybe take the opportunity to reformat this. Drop each item of the array on a new line.

File: test/unit/apps/companies/controllers/ukother.test.js [hunk 3]
```diff
@@ -147,7 +147,7 @@ describe('Company controller, uk other', function () {
           expect(getDisplayCompanyStub).to.be.calledWith(company)
           expect(res.locals).to.have.property('companyDetails')
           expect(res.locals).to.have.property('companyDetailsLabels')
-          expect(res.locals.companyDetailsDisplayOrder).to.deep.equal(['business_type', 'registered_address', 'alias', 'trading_address', 'uk_region', 'headquarter_type', 'sector', 'website', 'description', 'employee_range', 'turnover_range'])
+          expect(res.locals.companyDetailsDisplayOrder).to.deep.equal(['business_type', 'registered_address', 'trading_name', 'trading_address', 'uk_region', 'headquarter_type', 'sector', 'website', 'description', 'employee_range', 'turnover_range'])
```
Comment (tyom) [inline test/unit/apps/companies/controllers/ukother.test.js]: Reformat this as well to avoid long lines?

**Pull request merged**